src/state/api.js

```diff
@@ -5,6 +5,7 @@ const randomProbeURL = '__BASE_DOMAIN__/api/v1/probes/random/';
 const FETCH_ERROR_MESSAGES = {
   code5xx: 'Oh no! The server encountered an error.',
   code4xx: 'Oh no! Your client sent an invalid request.',
+  code404: 'No data found for the selected dimensions.'
```

Comment:
There's a new Prettier error on this line. It looks like it wants a trailing comma. (Trailing commas are a new default in Prettier 2.0.0.)

Reply:
How do you run the linter?

Reply:
You can run `npm run format` from the command line. Another option is to configure your editor to auto-run Prettier on all GLAM JS files. That's what I've set up.

Reply:
I wouldn't configure your editor to auto-run Prettier on Svelte files yet because it's got some bad known issues, including one that causes it to return garbage when Prettier 2.0.0+ is installed.

Reply:
https://github.com/UnwrittenFun/prettier-plugin-svelte/issues/84